specification/cognitiveservices/data-plane/LUIS/Programmatic/stable/v2.0/LUIS-Programmatic.json

```diff
@@ -2556,41 +2556,126 @@
         }
       }
     },
-    "/apps/{appId}/querylogs": {
+    "/api/v2.0/apps/{appId}/querylogs": {
       "get": {
-        "description": "Gets the query logs of the past month for the application.",
-        "operationId": "Apps_DownloadQueryLogs",
-        "parameters": [
-          {
-            "$ref": "#/parameters/AppIdInPath"
+          "description": "Gets the query logs of the past month for the application.",
+          "operationId": "Apps_DownloadApplicationQueryLogs",
+          "parameters": [
+              {
+                  "$ref": "#/parameters/AppIdInPath"
+              },
+              {
+                  "$ref": "#/parameters/IncludeResponseInPath"
+              }
+          ],
+          "responses": {
+              "200": {
+                "description": "A CSV file containing the query logs for the past month.",
+                "schema": {
+                  "type": "object",
+                  "format": "file",
+                  "description": "The query logs of an application for the past month in CSV format."
+                }
+              },
+              "default": {
+                "description": "Error Response.",
+                "schema": {
+                  "$ref": "#/definitions/ErrorResponse"
+                }
+              }
+          },
+          "produces": [
+              "application/octet-stream"
+          ],
+          "x-ms-examples": {
+              "Successful Download Application Query logs request": {
+                "$ref": "./examples/apps/SuccessfulDownloadApplicationQueryLogsRequest.json"
+              }
           }
-        ],
-        "responses": {
-          "200": {
-            "description": "A CSV file containing the query logs for the past month.",
-            "schema": {
-              "type": "object",
-              "format": "file",
-              "description": "The query logs of an application for the past month in CSV format."
-            }
+      }
+  },
+  "/api/v2.0/apps/{appId}/querylogsasync": {
+      "get": {
+          "description": "Gets the status of an asynchronous download query logs operation, and returns the logs if the operation completed.",
+          "operationId": "Apps_DownloadApplicationQueryLogsAsync",
```

Comment:
The operation ID must be unique. Change it to SendApplicationQueryLogsDownloadRequest for example